conda/recipes/pynvjitlink/meta.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2018-2024, NVIDIA CORPORATION.
+
+{% set data = load_file_data("pyproject.toml") %}
+
+{% if cuda_compiler_version is defined %}
+{% set cuda_major = cuda_compiler_version.split(".")[0]|int %}
+{% set cuda_minor = cuda_compiler_version.split(".")[1]|int %}
+{% else %}
+{% set cuda_major = 0 %}
+{% set cuda_minor = 0 %}
+{% endif %}
+
+package:
+  name: pynvjitlink
```

Comment:
```suggestion
  name: {{ project_data["name"] }}
```

Reply:
It seems like the name cannot be templated, likely due to some issue with evaluation order of Jinja variables in conda build.